zhaquirks/tuya/ts0601_valve.py

```diff
@@ -493,3 +502,97 @@ class GiexValveVar02(CustomDevice):
             }
         }
     }
+
+
+class GiexIrrigationStatus(t.enum8):
+    """Giex Irrigation Status Enum."""
+
+    Manual = 0x00
+    Auto = 0x01
+    Idle = 0x02
+
+
+(
+    TuyaQuirkBuilder("_TZE284_8zizsafo", "TS0601")  # Giex GX04
+    .tuya_battery(dp_id=59, power_cfg=TuyaPowerConfigurationCluster4AA)
+    .tuya_switch(
+        dp_id=1,
+        attribute_name="valve_one_on_off",
+        entity_type=EntityType.STANDARD,
+        translation_key="valve_one_on_off",
+        fallback_name="Valve 1",
+    )
+    .tuya_switch(
+        dp_id=2,
+        attribute_name="valve_two_on_off",
+        entity_type=EntityType.STANDARD,
+        translation_key="valve_two_on_off",
+        fallback_name="Valve 2",
+    )
+    .tuya_number(
+        dp_id=13,
+        attribute_name="valve_one_countdown",
+        type=t.uint16_t,
+        device_class=SensorDeviceClass.DURATION,
+        unit=UnitOfTime.MINUTES,
+        min_value=0,
+        max_value=1440,
+        step=1,
+        translation_key="valve_one_countdown",
+        fallback_name="Irrigation Time 1",
+    )
+    .tuya_number(
+        dp_id=14,
+        attribute_name="valve_two_countdown",
+        type=t.uint16_t,
+        device_class=SensorDeviceClass.DURATION,
+        unit=UnitOfTime.MINUTES,
+        min_value=0,
+        max_value=1440,
+        step=1,
+        translation_key="valve_two_countdown",
+        fallback_name="Irrigation Time 2",
```

Comment:
Due to [Home Assistant naming](https://developers.home-assistant.io/blog/2022/07/10/entity_naming/#:~:text=the%20rest%20of%20the%20words%20are%20lower%20case), this should be "Irrigation time 2"".
```suggestion
        fallback_name="Irrigation time 2",
```